recipes/sparrow/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build.cppstd import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=2.4"
+
+
+class SparrowRecipe(ConanFile):
+    name = "sparrow"
+    description = "C++20 idiomatic APIs for the Apache Arrow Columnar Format"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/man-group/sparrow"
+    topics = ("arrow", "apache arrow", "columnar format", "dataframe")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "use_date_polyfill": [True, False],
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "use_date_polyfill": False
```

Comment:
I see that this option is true by default in upstream's CMakeLists. Any insight into why this is turned off in the recipe?

Reply:
You are right, I just pushed a fix

Reply:
Btw it should fix the issue on OSX

Reply:
Does this mean that `use_date_polyfill` as `False` is not supported on Macos?